src/atmRequests.py

```diff
@@ -0,0 +1,78 @@
+import requests
+from models.atm import ATM
+import utils.constants
+from utils.exceptions import ATMValidationError, NessieApiError
+
+class ATMRequest(object):
+
+    def __init__(self, apiKey):
+        self.baseUrl = utils.constants.baseUrl
+        self.key = apiKey
+
+    def __buildParams(self, lat, lng, rad):
+        param = {'key': self.key}
+        if (lat != None):
+            param['lat'] = lat
+        if (lng != None):
+            param['lng'] = lng
+        if (rad != None):
+            param['rad'] = rad
+        return param
+
+    def __formatResponse(self, json):
+        respList = []
+        for atm in json['data']:
+            respList.append(ATM(atm))
+        return respList
+
+    def __validateParams(self, req):
+        lat = req.get('lat')
+        lng = req.get('lng')
+        rad = req.get('rad')
+
+        if (lat is not None or lng is not None or rad is not None):
+            if (lat is None or lng is None or rad is None):
+                raise ATMValidationError(utils.constants.missingFields)
+            if (lat < -90 or lat > 90 or lng < -180 or lng > 180 or rad <= 0):
```

Comment:
Instead of nesting two if statements under one if statement, would it be better to do a `if statement to check for any None's` then do a `elif statement for the invalid fields`